Apps/WebClient/src/ClientApp/src/components/report/medicationHistory.vue

```diff
@@ -1,19 +1,30 @@
 <script lang="ts">
+import { saveAs } from "file-saver";
 import Vue from "vue";
-import { Component, Emit, Prop, Ref, Watch } from "vue-property-decorator";
+import { Component, Emit, Prop, Watch } from "vue-property-decorator";
 import { Action, Getter } from "vuex-class";
 
 import ProtectiveWordComponent from "@/components/modal/protectiveWord.vue";
-import ReportHeaderComponent from "@/components/report/header.vue";
 import { DateWrapper } from "@/models/dateWrapper";
 import MedicationStatementHistory from "@/models/medicationStatementHistory";
+import PatientData from "@/models/patientData";
 import ReportField from "@/models/reportField";
 import ReportFilter from "@/models/reportFilter";
 import User from "@/models/user";
 import { SERVICE_IDENTIFIER } from "@/plugins/inversify";
 import container from "@/plugins/inversify.container";
-import { ILogger } from "@/services/interfaces";
-import PDFUtil from "@/utility/pdfUtil";
+import { ILogger, IReportService } from "@/services/interfaces";
+
+import { ReportType, TemplateType } from "../../models/reportRequest";
+
+interface ReportHeader {
```

Comment:
note. This might need to be moved to a common file since is going to be shared among the other reports